src/biotite/structure/graphics/rna.py

```diff
@@ -0,0 +1,260 @@
+# This source code is part of the Biotite package and is distributed
+# under the 3-Clause BSD License. Please see 'LICENSE.rst' for further
+# information.
+
+__name__ = "biotite.structure.graphics"
+__author__ = "Tom David Müller"
+__all__ = ["plot_nucleotide_secondary_structure"]
+
+import shutil
+import numpy as np
+import matplotlib.pyplot as plt
+from .. import pseudoknots
+from ...application.viennarna import RNAplotApp
+
+
+def plot_nucleotide_secondary_structure(
+    base_labels, base_pairs, length, layout_type=RNAplotApp.Layout.NAVIEW,
+    draw_pseudoknots=True, pseudoknot_order=None, angle=0, bond_linewidth=1,
+    bond_linestyle=None, bond_color='black', backbone_linewidth=1,
+    backbone_linestyle='solid', backbone_color='grey', base_font=None,
+    base_box=None, annotation_positions=None, annotation_offset=8.5,
+    annotation_font=None, border=0.03, bin_path="RNAplot"
+    ):
+    """
+    Generate 2D plots of nucleic acid secondary structures using the
+    interface to *RNAplot*, which is part of the *ViennaRNA* software
+    package.
+
+    Internally a :class:`biotite.application.viennarna.RNAplotApp` 
+    instance is created to generate coordinates for each individual base 
+    on a 2D plane. *ViennaRNA* must be installed in order to use this 
+    function.
+
+    Parameters
+    ----------
+    base_labels : iterable
+        The labels denoting the type of each base.
+    base_pairs : ndarray, shape=(n,2)
+        Each row corresponds to the positions of the bases in the
+        sequence. The positions are counted from zero.
+    length : int
+        The number of bases in the sequence.
+    layout_type : RNAplotApp.Layout, optional (default: RNAplotApp.Layout.NAVIEW)
+        The layout type according to the *RNAplot* documentation.
+    draw_pseudoknots : bool, optional (default: True)
+        Whether pseudoknotted bonds should be drawn.
+    pseudoknot_order : iterable, optional (default: None)
+        The pseudoknot order of the input `base_pairs`. If no pseudoknot
```

Comment:
For clarification: Could you write `The pseudoknot order of each base in the input...` ?